qpmodel/Statis.cs

```diff
@@ -346,7 +346,8 @@ public void ComputeStats(int index, List<Row> samples)
                 foreach (var g in groups)
                     sortgroup.Add(g.Key, g.Count());
 
-                var sorted = from pair in sortgroup orderby pair.Value descending select pair;
+                // use top 100 values to calculate frequency, ensure that pairs are sorted in a fixed order.
+                var sorted = from pair in sortgroup orderby pair.Value descending, pair.Key descending select pair;
```

Comment:
we must keep the order of @sorted. Sort by @pair.Value, and then sort by @pair.Key. Otherwise top 100 of @sorted cant not be guaranteed.